### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by aliakhraisha**
-  Build the "Works" section
-  Build the "About myself" section (placed after the projects and before the contact form)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -22,48 +28,192 @@ h1 {
...
+
+#frameworks {
+  background: #e2505c;
+  background-image: url("images/ic_framework.png");
+}
+
+#skills {
+  background: #febf52;
+  background-image: url("images/ic_skills.svg");
 }
```
Comment (cynthiainga) [inline styles.css:217]: - [ ] Good job by implementing the language section but you still have some issue to fix with your design. Between the languages there is space, please kindly add the padding between your languages. Your design should stick as much as possible to the Figma design 🙃 🙃

Your design

Screen Shot 2022-10-01 at 20 25 12

Figma design

Screen Shot 2022-10-01 at 20 24 36

File: index.html [hunk 2]
```diff
@@ -14,18 +14,129 @@
...
     <footer>
     </footer>
-
+</section>
+<section id="works">
+    <h2>Projects</h2>
+    <section id="project">
+      <img id="projectimage" src="images/Rectangle 21.1.svg" alt="first project image">
+      <div id="projectdetails">
+        <div id="projectname"><p >Project name goes here</p></div>
```
Comment (cynthiainga) [inline index.html:34]: - [ ] For the good practice of HTML, I think the name of the project is a title and not a paragraph. Please kindly change your `<p>` tag to the `<hx>` tag 🙃 🙃  also made the change to all the `projectname` 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @aliakhraisha, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Used correct Github flow ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (aliakhraisha) [reply styles.css:217]: design modified to match the figma

File: index.html (hunk 2 above)
Comment (aliakhraisha) [reply index.html]: all <p>  tag  of the projects  names changed to  <hx>

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @aliakhraisha , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**